Wikipedia/assets/codemirror/codemirror-index.html

```diff
@@ -819,6 +819,23 @@
       focus: () => {
         editor.focus()
       },
+      focusWithoutScroll: () => {
+        const scrollY = window.scrollY
+        editor.focus()
+        window.scrollTo(window.scrollX, scrollY)
+      },
+      selectLastFocusedMatch: () => {
+        const state = editor.state.search
+        if (!state) {
+          return
+        }
+        const from = state.posFrom
+        const to = state.posTo
+        if (!from || !to) {
+          return
+        }
+        editor.setSelection(from, to)
```

Comment:
May want the `scroll` param with `setSelection` here just in case code mirror is doing some internal scroll-y bits...
``editor.setSelection(from, to, {scroll: false})``

Reply:
Could also try `editor.setCursor(from)` here... might prevent the "cut copy paste" menu from appearing... though `editor.setSelection(from, null, {scroll: false})` might work for that too...

haha we have many options... none of them good 😂